src/Reports/ElementsInUseReport.php

```diff
@@ -83,7 +83,7 @@ public function columns()
      * @param BaseElement $item
      * @return string
      */
-    protected function getEditLink($value, $item)
+    protected function getEditLink($value, BaseElement $item)
```

Comment:
If we can removed this change then LGTM - this would be a semver violation for a patch change

Reply:
Oh but this module isn't stable in this new major release line yet, so it's OK